features/gba-dev/fields-mapping-gba.csv

```diff
@@ -53,7 +53,7 @@ verblijfplaats.verblijfadres.postcode,verblijfplaats.straat|verblijfplaats.postc
 verblijfplaats.verblijfadres.regel1,verblijfplaats.straat|verblijfplaats.locatiebeschrijving|verblijfplaats.land|verblijfplaats.regel1
 verblijfplaats.verblijfadres.regel2,verblijfplaats.straat|verblijfplaats.locatiebeschrijving|verblijfplaats.land|verblijfplaats.regel2
 verblijfplaats.verblijfadres.regel3,verblijfplaats.straat|verblijfplaats.locatiebeschrijving|verblijfplaats.land|verblijfplaats.regel3
-verblijfplaats.verblijfadres.straat,verblijfplaats.straat|verblijfplaats.naamOpenbareRuimte|verblijfplaats.locatiebeschrijving|verblijfplaats.land
+verblijfplaats.verblijfadres.straat,verblijfplaats.naamOpenbareRuimte|verblijfplaats.locatiebeschrijving|verblijfplaats.land
```

Comment:
verblijfadres.straat is ten onrechte verwijderd. die is nodig om verblijfplaats type te bepalen

Reply:
Aangepast